Dockerfile

```
@@ -4,9 +4,12 @@ ARG version=22.04
 FROM ubuntu:"${version}"
 ARG DEBIAN_FRONTEND=noninteractive
 
-# Set the user ID to the default value of 1001 since different ubuntu
-# versions yield to different user IDs with `useradd`
-ENV USER_ID=1001
+# Deterministic UID (first user). Helps with docker build cache
+ENV USER_ID=1000
+# Delete the default ubuntu user and group in Ubuntu 23.04+
+# that conflicts with linuxbrew i.e UID=1000 GID=1000
+# shellcheck disable=SC2015
+RUN touch /var/mail/ubuntu && chown ubuntu /var/mail/ubuntu && userdel -r ubuntu || true
```

```suggestion
RUN ( touch /var/mail/ubuntu && chown ubuntu /var/mail/ubuntu && userdel -r ubuntu ) || true
```
or
```suggestion
RUN touch /var/mail/ubuntu; chown ubuntu /var/mail/ubuntu; userdel -r ubuntu || true
```
or something? Would like to avoid disabling the `shellcheck` here if possible. Can try different versions locally with `brew style` before commiting/deciding.